src/main/drivers/serial_uart_hal.c

```diff
@@ -204,8 +214,17 @@ void uartTryStartTxDMA(uartPort_t *s)
             return;
         }
 
+#if defined(STM32G4)
+        // XXX Explain why this test is required for G4
```

Comment:
Do you want to add an explanation here? If not the comment can be removed as we won't know what to put here. :wink:

Reply:
It's unknown yet. I'd leave it for future investigation/reference...

Reply:
As the original PR's code disabled EN bit checking as explained in https://github.com/betaflight/betaflight/pull/9407#discussion_r373286676, it was necessary to check active on going DMA transfer this way. Since the EN bit checking is back again, this section is not necessary anymore.